app/components/frame-viewer.cjsx

```diff
@@ -193,23 +195,24 @@ module.exports = React.createClass
     unless @state.panEnabled
       @setState panEnabled: true, =>
         this.refs.subjectImage.focus()
-  
-  togglePanOff: ->
-    @setState panEnabled: false
-  
+
+  togglePanOff: (e) ->
+    unless e.relatedTarget?.title is "zoom out" or e.relatedTarget?.title is "zoom in"
```

Comment:
hey Jim, i think i pushed an update while you were reviewing, sorry about that! yes Andrea and i are trying to fix a few little things across browsers, i updated the Pull Request title with [WIP] to better note it's still a work in progress...